__tests__/app.test.js

```diff
@@ -138,21 +138,55 @@ describe("/api/articles", () => {
                 });         
         });
     });
-    test("GET: 200 - articles are ordered by created_at in descending order", ()=>{
+    test("GET: 200 - articles are ordered by created_at in descending order by default", ()=>{
         return request(app)
         .get("/api/articles?sort_by=created_at")
         .expect(200)
         .then(({body}) => {
             expect(body.articles).toBeSortedBy("created_at", { descending: true})
         })
     });
-    test('GET: 200 - articles can be ordered ascending', () => {
+    test("GET: 200 - articles can be ordered ascending", () => {
         return request(app)
           .get('/api/articles?order=asc')
           .expect(200)
           .then(({ body }) => {
             expect(body.articles).toBeSortedBy('created_at');
-          });
+        });
+    });
+    test("GET: 200 - orders by default if sort_by is misspelt", () => {
+        return request(app)
+          .get('/api/articles?soort_by=title')
+          .expect(200)
+          .then(({body}) => {
+            expect(body.articles).toBeSortedBy("created_at", { descending: true})
+        })
+    });
+    test("GET: 200 - orders by default if order query is misspelled", () => {
+        return request(app)
+          .get('/api/articles?orrder=asc')
+          .expect(200)
+          .then(({body}) => {
+            expect(body.articles).toBeSortedBy("created_at", { descending: true})
+        })
+    test("GET: 404 - responds with an error when passed an article_id that is valid but not present in our database", ()=>{
+        return request(app)
+        .get("/api/articles?article_id=9999")
```

Comment:
This is testing against a different endpoint. This endpoint is for an array of articles, but this test and the one below are requesting for an individual article.